### PR TITLE
ST configuration directory if syncthing binary is a snap

### PR DESCRIPTION
**Pull request by timski**
That modification let's ST-GTK find the correct configuration folder if the syncthing binary is a snap. My first ever python code. :)

### CODE REVIEW

**Comment by kozec** [comment]
Three problems, in order of severity :)

- 1st run wizard is started only on 1st run, so this is not gonna work after restart.
- globals are bad, it may be better to store this in configuration. Just 
- what's up with all that removed whitespace?

File: syncthing_gtk/wizard.py [hunk 1]
```diff
@@ -343,6 +345,10 @@ def search(self):
 					# File exists and is executable, run it and parse
 					# version string from output
 					log.info("Binary found in %s", bin_path)
+					if path == '/snap/bin':
+						set_is_snap()
```
Comment (kozec) [inline syncthing_gtk/wizard.py]: You can just add new key [somewhere here](https://github.com/syncthing/syncthing-gtk/blob/886466470e2c590352e9f4ab5f6bf104311be9a6/syncthing_gtk/configuration.py#L45) and then do `self.config['syncthing_is_snap'] = True` here. Config is "global" in sense that you can create instantiate Configuration object anywhere and read settings from it.

**Comment by funkyfuture** [comment]
unless i'm missing a point (haven't looked deeply), any change of Syncthing's configuration should be done via [its API](https://docs.syncthing.net/rest/system-config-post.html) and not directly on disk. with this in place, it wouldn't matter how syncthing was installed, rendering this pr obsolete.

**Comment by funkyfuture** [comment]
wouldn't a widget to enter an api-key by hand be feasible as an alternative?

**Comment by kozec** [comment]
It would be much clunkier than next>next>next>finish dialog that generates that key :)

**Comment by st-review** [comment]
:robot: *beep boop*

I'm going to close this pull request as it has been idle for more than 90 days.

This is not a rejection, merely a removal from the list of active pull requests that is periodically reviewed by humans. The pull request can be reopened when there is new activity, and merged once any remaining issues are resolved.

**Pull request closed without merging**